JavaScriptTasks/Session 3 - Js_DynamicScope(4-03-2020)/CanYouKeepASecret.js

```diff
@@ -0,0 +1,23 @@
+// PROBLEM STATEMENT :Can you keep a secret
+
+// http://www.codewars.com/kata/can-you-keep-a-secret 
+
+// SOLUTION:
+
+function createSecretHolder(secret) {
+
+  var obj = {
+  
+  SecretKey : secret ,
```

Comment:
Remove this from object and add as a variable in function scope

Reply:
modified the code as :

function createSecretHolder(secret) {
  let SecretKey = secret ;
  var obj = {
  getSecret :function () { 
      return SecretKey;
    }
   setSecret :function (x) { 
      SecretKey= x;
        }
    }
    return obj;
  }